notebooks/tutorials/landscape_evolution/space/sediment_flux_map.eps

```diff
@@ -0,0 +1,3689 @@
+%!PS-Adobe-3.0 EPSF-3.0

```

Comment:
I believe this is also an output file generated by a notebook and so shouldn't be under version control.

Reply:
done, deleted.